recipe/meta.yaml

```diff
@@ -23,45 +22,57 @@ build:
   ignore_run_exports:
     - icu
     - expat
+    - jpeg  # [win]
 
 requirements:
   build:
     - {{ compiler('c') }}
-    - automake
-    - libtool
-    - gettext
+    - automake      # [not win]
+    - libtool       # [not win]
+    - gettext       # [not win]
     - pkg-config
-    - make
-    - patch     # [unix]
-    - m2-patch  # [win]
+    - make          # [not win]
+    - patch         # [unix]
+    - m2-patch      # [win]
+    - cmake         # [win]
+    - ninja-base    # [win]
   host:
     - expat
-    - fontconfig
+    - fontconfig    # [not win]
     - freetype
     - icu
     - jpeg
-    - libiconv  # [osx]
+    - libiconv      # [osx]
     - libpng
     - libtiff
-    - libwebp-base
+    - libwebp-base  # [not win]
```

Comment:
Do we need to enable **webp** in `build.sh`?

Reply:
The check is set to auto by default. It is found correctly:
 ```
                 INFO        Support for gd/gd2 images:        no                                                                                                                                                                           
                  INFO        Support for Zlib:                 yes                                                                                                                                                                          
                  INFO        Support for PNG library:          yes                                                                                                                                                                          
                  INFO        Support for JPEG library:         yes                                                                                                                                                                          
                  INFO        Support for WebP library:         yes                                                                                                                                                                          
                  INFO        Support for HEIF library:         no                                                                                                                                                                           
                  INFO        Support for AVIF library:         no                                                                                                                                                                           
                  INFO        Support for TIFF library:         yes                                                                                                                                                                          
                  INFO        Support for Freetype 2.x library: yes                                                                                                                                                                          
                  INFO        Support for Fontconfig library:   yes                                                                                                                                                                          
                  INFO        Support for Xpm library:          no                                                                                                                                                                           
                  INFO        Support for liq library:          no                                                                                                                                                                           
                  INFO        Support for complex text:         no                                                                                                                                                                           
                  INFO        Support for pthreads:             yes 
```